frontend/src/components/ui/MicButton.tsx

```
@@ -0,0 +1,63 @@
+  const label = isRecording ? 'End call' : 'Start call'
+  return (
+    <button
+      onClick={onClick}
+      disabled={disabled}
+      className={cn(
+        micButtonVariants({ status, size }),
+        disabled && 'cursor-not-allowed opacity-70',
+        className,
+      )}
+      aria-label={label}
+    >
```

Biome (2.4.4)

[error] Provide an explicit type prop for the button element.

(lint/a11y/useButtonType)